presto-product-tests/bin/product-tests-suite-3.sh

```diff
@@ -19,4 +19,9 @@ presto-product-tests/bin/run_on_docker.sh \
     -g storage_formats,cli,hdfs_impersonation,authorization \
     || exit_code=1
 
+presto-product-tests/bin/run_on_docker.sh \
+    singlenode-ldap-bind-dn \
+    -g ldap -x simba_jdbc \
+    || exit_code=1
```

Comment:
add this to `product-tests-suite-7-non-generic.sh` rather than suite 3.

there is no point in running these tests for various hadoop distros